### DISCUSSION TITLE
Converting permeability and ER in presence/absence of BCRP inhibitor into Enzyme Parameters

### DISCUSSION THREAD

**Discussion by cbdoc** [Open-Systems-Pharmacology/Forum#951] [Questions & Problems]
I'm trying run a mouse simulation of a small molecule that exhibits EHC, but so far have not been able to model EHC. 

- Simulation > Parameters > Enzymes > ABCG2 (BCRP) added: **I'm unsure what parameters to input there so left the default values. I have data that shows % permeability and ER in presence and absence of BCRP inhibitor, but that's it.**

**Comment by StephanSchaller** [comment]
@cbdoc, you need to add the EHC Event, otherwise there is no release of bile into the gut:

When you create the Simulation, make sure to create an event for EHC:
![image](https://user-images.githubusercontent.com/25477643/129558366-365233af-620d-4cd2-86d5-4d0b70635c23.png)
